app/timetables/models.py

```diff
@@ -175,3 +175,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Vendor(SlugifyMixin, TimestampMixin):
+    """Model representing food service-provider"""
+
+    name = models.CharField(max_length=255)
+    slug = models.SlugField(max_length=255, unique=True, editable=False)
```

Comment:
@andela-kerinoso In that case, we could add a more verbose error message other than `raise ValidationError("This object already exists.")`, because technically, an exact duplicate object doesn't yet exist. We could use something like `That vendor name is taken` ... Something more user-friendly along these lines. What do you think? It might be out of scope for this ticket but it's something we should look into.